extensions/json-language-features/package.nls.json

```diff
@@ -7,7 +7,7 @@
 	"json.schemas.fileMatch.item.desc": "A file pattern that can contain '*' to match against when resolving JSON files to schemas.",
 	"json.schemas.schema.desc": "The schema definition for the given URL. The schema only needs to be provided to avoid accesses to the schema URL.",
 	"json.format.enable.desc": "Enable/disable default JSON formatter",
-	"json.tracing.desc": "Traces the communication between VS Code and the JSON language server.",
+	"json.tracing.desc": "Traces the communication between Azure Data Studio and the JSON language server.", // {{SQL CARBON EDIT}} Change product name to ADS
```

Comment:
does this cause any issues? the actual spec for json does not allow comments, jsonc is required for that and not all json readers support it.

Reply:
yeah you can see the packaged task is failing for not being able to properly read the json

Reply:
Yeah wasn't sure if that was actually going to cause problems. Will fix